language.md

```diff
@@ -119,4 +119,6 @@ Nếu bạn nói tiếng Việt, [nhấn vào link này](vietnamese/kechuyen.md)
 
 Se vi parolas esperanton, alklaki [tie] (esperanto/komenci.md).
 
-ئەگەر ئۇيغۇرچە سۆزلىسىڭىز، [بۇ يەرگە كىرىڭ](uyghurish/uyghurish-story.md)
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.